virttest/utils_test/qemu/__init__.py

```diff
@@ -204,10 +204,20 @@ def _check_driver_stat():
 
     for drv in driver.split():
         error_context.context("Check %s driver state." % drv, LOG.info)
+        driver_svc_map = {
+            "viorng": "VirtRng",
+            "vioser": "VirtioSerial",
+            "viofs": "VirtioFsDrv",
+            "vioinput": "VirtioInput",
+        }
+        driver_svc = driver_svc_map[drv] if drv in driver_svc_map.keys() else drv
```

Comment:
Shall we consider moving this to https://github.com/avocado-framework/avocado-vt/blob/master/virttest/utils_windows/__init__.py? as a universal configuration?